arangod/RocksDBEngine/RocksDBIndexFactory.cpp

```diff
@@ -403,6 +455,7 @@ RocksDBIndexFactory::indexAliases() const {
   return {
       {"hash", "persistent"},
       {"skiplist", "persistent"},
+      {"mdi", "zkd"},
```

Comment:
The aliases in this object are `{ old name, new name }`. So here we would add `mdi` as an alias for `zkd`.
Not sure if this is intentional or if it should be the other way round. Potentially it is ok for this PR but can be changed later if zkd is renamed to mdi.

Reply:
that was my plan